packages/cli/src/util/loadMetroConfig.js

```diff
@@ -90,7 +89,7 @@ export default async function load(
   ctx: ContextT,
   // $FlowFixMe - troubles with empty object being inexact
   options?: ConfigOptionsT = {}
-): Promise<ConfigT> {
+) {
```

Comment:
Why you removed type signature here?

Reply:
because Flow sees `ConfigT` as `any`, for some reason it's unable to get the types from `metro-config` even though they have `.js.flow` files. Didn't investigate though